thunder/examine/__init__.py

```diff
@@ -14,6 +15,13 @@
 from warnings import warn
 from itertools import chain
 
+try:
```

Comment:
you can use https://github.com/Lightning-AI/utilities/#3-general-python-utilities-lightning_utilitiescore